pkg/k8s/watchers/cilium_endpoint.go

```diff
@@ -262,3 +260,14 @@ func CreateCiliumEndpointLocalPodIndexFunc() cache.IndexFunc {
 		return indices, nil
 	}
 }
+
+// Helpers...
+
+func (k *K8sWatcher) updatePromMX(cep *types.CiliumEndpoint) {
+	if cep == nil {
+		return
+	}
+	if p := k.endpointManager.LookupPodName(k8sUtils.GetObjNamespaceName(cep)); p != nil {
+		metrics.EndpointPropagationDelay.WithLabelValues().Observe(time.Since(p.GetCreatedAt()).Seconds())
```

Comment:
Do we need to account for scheduler delay? I believe so; pods may be in Pending state for a long time, but AIUI endpoints are created only when a pod is actually scheduled and executed.

In other words, we don't actually want to capture the full pod scheduling delay, right?